include/swift/Basic/Features.def

```diff
@@ -410,7 +410,7 @@ EXPERIMENTAL_FEATURE(SafeInterop, true)
 EXPERIMENTAL_FEATURE(AssumeResilientCxxTypes, true)
 
 // Isolated deinit
-SUPPRESSIBLE_LANGUAGE_FEATURE(IsolatedDeinit, 371, "isolated deinit")
+EXPERIMENTAL_FEATURE(IsolatedDeinit, true)
```

Comment:
I believe this line should be

```
SUPPRESSIBLE_EXPERIMENTAL_FEATURE(IsolatedDeinit, false)
```

Reply:
You're right my mistake here, let me change that